src/App.css

```diff
@@ -7,16 +7,13 @@
 }
 
 .dashboard {
-  max-width: 50%;
+ width: 50%;
```

Comment:
Det kan være en ide å sette til en @media stil også med max-width på 768px for å håndtere litt mindre skjermer.